app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -65,7 +65,8 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   private val context: Context,
   private val viewModelProvider: ViewModelProvider<ProfileChooserViewModel>,
   private val profileManagementController: ProfileManagementController,
-  private val oppiaLogger: OppiaLogger
+  private val oppiaLogger: OppiaLogger,
+  private val adapterBuilder: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
For consistency: `private val multiTypeBuilderFactory: `

Reply:
updated.